src/expect_interface.rs

```diff
@@ -182,15 +183,15 @@ impl<'a> ExpectGrpcCall<'a> {
         }
     }
 
-    pub fn returning(&mut self, token_id: Option<u32>) -> &mut Tester {
+    pub fn returning(&mut self, res: Result<u32, Status>) -> &mut Tester {
```

Comment:
Don't think this should be a `Result` or then a `Option<Result>`... didn't look to closely into this particular expect, but the pattern was (tho arguably confusing), that it'd only validate when a `Some` was provided, while `None` would mean: while I expect this call, I don't care what it gets called with...

Reply:
Nevermind that's the return